wgpu-core/src/command/clear.rs

```diff
@@ -338,6 +338,11 @@ fn clear_texture_via_buffer_copies<A: HalApi>(
         hal::FormatAspects::COLOR
     );
 
+    if texture_desc.format == wgt::TextureFormat::NV12 {
+        // TODO: Currently COPY_DST for NV12 textures is unsupported.
```

Comment:
Should this have a validation error? Or is this something you'd need ycbcr samplers for in Vulkan and that hasn't been implemented yet.

Reply:
This code is the runtime clear stuff, which we unconditionally invoke right now. So basically we can't properly zero them out right now.